examples/pong_tutorial_01/main.rs

```diff
@@ -27,9 +27,13 @@ fn main() -> amethyst::Result<()> {
         env!("CARGO_MANIFEST_DIR")
     );
 
+    // This line is not mentionned in the pong tutorial as it is specific to the context
```

Comment:
"mentioned"

Reply:
@Moxinilian latest commit missed this one :P

Reply:
damn it